src/referencePackages/src/microsoft.bcl.asyncinterfaces/1.1.0/Microsoft.Bcl.AsyncInterfaces.1.1.0.csproj

```diff
@@ -1,8 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
-  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))/dir.props" />
```

Comment:
I think in general we don't want to check-in changes to projects that have already been added to SBRP, since they usually have required some work to get to a buildable state - unless there's a particular reason these have all been touched. I think this applies to most other projects in this PR.